extensions.json

```diff
@@ -664,6 +664,12 @@
       "repository": "https://github.com/JeffersonQin/VSCode-LaTeX-Snippets",
       "version": "1.2.3"
     },
+    {
+      "id": "jebbs.plantuml",
+      "version": "2.13.12",
```

Comment:
I am not sure if this field does what I hope it does. I hope it would tell the build service to start with that version of the extension, and then continue publishing more recent ones as well?

Reply:
By declaring the version, the script checks whether the version in the package.json matches that version. It is not updated automatically.

The latest version seems to be 2.14.1 (`master` branch), but the latest tag is `v2.14.0`. I'd propose to follow the tags and add `"checkout": "v2.14.0"`, and to adapt the version accordingly.

Reply:
> By declaring the version, the script checks whether the version in the package.json matches that version. It is not updated automatically.

Slight precision here: `extension.version` is only compared against what is already on OpenVSX: https://github.com/open-vsx/publish-extensions/blob/994cbe51fcabac973de54445f15344804a6f8cd5/publish-extensions.js#L65-L74

- if `extension.version` is **older** than what's on OpenVSX, the script throws an error (i.e. we're trying to publish something older than what's on OpenVSX, which likely means someone else took over the publishing & we should stop doing that here)
- if `extension.version` is **equal** to what's on OpenVSX, this extension is simply skipped (no need to re-publish the same version again)
- if `extension.version` is **newer** than what's on OpenVSX (or if there is nothing on OpenVSX, or if `extension.version` isn't specified), we go ahead and try to publish

So the version specified in the extension's `package.json` is never checked by our script (however, if the actual `package.json` version is equal or older to what's on OpenVSX, publishing will still fail but at a later point, i.e. in `ovsx` CLI itself)

Reply:
Thanks for the clarification!

> if the actual package.json version is equal or older to what's on OpenVSX, publishing will still fail but at a later point, i.e. in ovsx CLI itself

Slight precision here: ovsx allows to publish older version. It fails only if the exact version is already published.

Reply:
> I'd propose to follow the tags and add "checkout": "v2.14.0", and to adapt the version accordingly.

Done.

Reply:
I think it would be awesome to have a way to request the build service to build a "one-off", missing version of an extension it already builds. For many reasons it can be desirable to use a specific, older version, of an extension. 

Maybe this could be done by submitting a PR to an alternate branch, that would upon merge trigger the build service for whatever extension(s) have changed?